slides/intro.md

```diff
@@ -0,0 +1,157 @@
+---
+theme: csc-2019
+lang: en
+pagetitle: Introduction
+---
+
+# Kielipankki & CSC {.title}
+
+Corpora, language technology, computational linguistics, corpus linguistics, supercomputers and all that
+
+<div class="column">
+![](https://mirrors.creativecommons.org/presskit/buttons/88x31/png/by-sa.png)
+</div>
+<div class="column">
+<!-- <small> -->
+<!-- All materials (c) 2023 by CSC – IT Center for Science Ltd. -->
+<!-- This work is licensed under a **Creative Commons Attribution-ShareAlike** 4.0 -->
+<!-- Unported License, [http://creativecommons.org/licenses/by-sa/4.0/](http://creativecommons.org/licenses/by-sa/4.0/) -->
+<!-- </small> -->
+</div>
+
+# Kielipankki <img src="img/kielipankki_logo.png" style="display:inline-block;position:absolute;top:-20px;">
+
+## AKA The Language Bank of Finland
+
+* We create, solicit and receive corpora
+
+* We enrich the corpora with NLP tools
+
+* We make available our enriching tools, plus others
+
+* Hundreds of corpora, tens of billions of words
```

Comment:
If I didn't know better, just reading this would make me think that we only hoard corpora but don't provide access to anything besides tools. Unless soliciting can be offering in non-sexwork contexts too, but I suppose not?

Might be good to make the providing side clear too.

Reply:
Added an extra bullet for that